stars/test/test_planet.py

```diff
@@ -87,7 +87,7 @@ def test_habitability2(self):
         r = race.Race(hab_gravity=0, hab_gravity_stop=100,
             hab_temperature=0, hab_temperature_stop=100,
             hab_radiation=0, hab_radiation_stop=100)
-        self.assertEqual(p.habitability(r), 41)
+        self.assertEqual(p.habitability(r), 41)# shouldent this be 66%?
```

Comment:
Is it now throwing an error or are you saying it should be 66% based on manual calculations?

Reply:
Its because I didn't understand the equation for the habitability. It is very complex and annoying but 41 is correct.

Reply:
```suggestion
        self.assertEqual(p.habitability(r), 41)
```